static/app/components/modals/widgetViewerModal.tsx

```diff
@@ -323,38 +342,71 @@ function WidgetViewerModal(props: Props) {
           </Container>
         )}
         {widget.queries.length > 1 && (
-          <React.Fragment>
-            <Alert type="info" icon={<IconInfo />}>
-              {t(
-                'This widget was built with multiple queries. Table data can only be displayed for one query at a time.'
-              )}
-            </Alert>
-            <StyledSelectControl
-              value={selectedQueryIndex}
-              options={queryOptions}
-              onChange={(option: SelectValue<number>) => {
-                router.replace({
-                  pathname: location.pathname,
-                  query: {
-                    ...location.query,
-                    [WidgetViewerQueryField.QUERY]: option.value,
-                    [WidgetViewerQueryField.PAGE]: undefined,
-                    [WidgetViewerQueryField.CURSOR]: undefined,
-                  },
-                });
-
-                trackAdvancedAnalyticsEvent(
-                  'dashboards_views.widget_viewer.select_query',
-                  {
-                    organization,
-                    widget_type: widget.widgetType ?? WidgetType.DISCOVER,
-                    display_type: widget.displayType,
-                  }
-                );
-              }}
-            />
-          </React.Fragment>
+          <Alert type="info" icon={<IconInfo />}>
+            {t(
+              'This widget was built with multiple queries. Table data can only be displayed for one query at a time.'
+            )}
+          </Alert>
         )}
+        <StyledSelectControl
+          value={selectedQueryIndex}
+          options={queryOptions}
+          onChange={(option: SelectValue<number>) => {
+            router.replace({
+              pathname: location.pathname,
+              query: {
+                ...location.query,
+                [WidgetViewerQueryField.QUERY]: option.value,
+                [WidgetViewerQueryField.PAGE]: undefined,
+                [WidgetViewerQueryField.CURSOR]: undefined,
+              },
+            });
+
+            trackAdvancedAnalyticsEvent('dashboards_views.widget_viewer.select_query', {
+              organization,
+              widget_type: widget.widgetType ?? WidgetType.DISCOVER,
+              display_type: widget.displayType,
+            });
+          }}
+          components={{
+            // Replaces the displayed selected value
+            SingleValue: containerProps => {
+              return (
+                <components.SingleValue
+                  {...containerProps}
+                  // Overwrites some of the default styling that interferes with highlighted query text
+                  getStyles={() => ({wordBreak: 'break-word'})}
```

Comment:
I'm noticing that the words are breaking earlier than I expected:
<img width="1080" alt="Screen Shot 2022-03-22 at 3 46 04 PM" src="https://user-images.githubusercontent.com/22846452/159563380-9ae42c8c-b018-48d5-b083-7d20bedf3fdd.png">

Locally I added `flex: 1` through the devtools and it fixed it, do you see the same and can we add that?

Reply:
Another styling thing. We added `max-height: 40px` before for snapshot tests on line 681 but it's causing the field to lose its padding with the table. Can we get rid of it? I'm wondering why the snapshot test looks so wonky but now that I think of it, it doesn't feel great to add code that only satisfies the snapshot test

https://user-images.githubusercontent.com/22846452/159564357-d3beafc1-1991-43d1-9aec-d8bb7cd1b2a8.mov

Reply:
@narsaynorath Hmm I wasn't able to replicate the words breaking earlier, but I added `flex: 1` anyways since it didn't have any adverse effects. (I think we're also on the same browser)

I removed `max-height: 40px` so padding should be fixed between the table now

Also added an IconSearch to the query selector since it was looking empty and added a default label for queries with empty conditions and no alias. Latest screenshots are updated in the PR description!